pkg/sippyserver/server.go

```diff
@@ -935,7 +998,10 @@ func (s *Server) Serve() {
 		serveMux.HandleFunc("/api/releases/health", s.jsonReleaseHealthReport)
 		serveMux.HandleFunc("/api/releases/tags", s.jsonReleaseTagsReport)
 		serveMux.HandleFunc("/api/releases/pull_requests", s.jsonReleasePullRequestsReport)
-		serveMux.HandleFunc("/api/releases/job_runs", s.jsonReleaseJobRunsReport)
+		serveMux.HandleFunc("/api/releases/job_runs", s.jsonListPayloadJobRuns)
+
+		serveMux.HandleFunc("/api/releases/{release}/{stream}/{arch}/analysis",
```

Comment:
This could easily fit into the release health API we have today, but it's a little slow (1.5s), and the release health is called multiple times on startup when we refresh metrics, slowing down development quite a bit.

Reply:
I'm a little concerned about introducing any gorilla packages, given they've been searching for maintainers for a long time.  https://github.com/gorilla/mux/issues/659

It may not be as pretty but this is also deviating from the existing API's which use query params for just about everything.

Reply:
That's too bad about gorilla, I can remove.

I probably read into our discussion about moving to things like test IDs and assumed that included moving to (I think) more correct URL structures and less reliance on query params. I can change that too but at some point we should talk about improving those.